src/components/components/ReinforcementStatement/ReinforcementStatement.scss

```diff
@@ -0,0 +1,44 @@
+.Reinforcement {
+  display: flex;
+  align-items: center;
+  margin: 96px;
+  gap: 96px;
+
+  &__image {
+    border-radius: 8px;
+  }
+
+  &Reinforcement__statement {
+
+    &__header {
+      color: var(--color-bunting);
+      font-weight: 700;
+      line-height: 133%;
+      margin: 0;
+      margin-bottom: 0;
+    }
+
+    &__underline {
+      display: inline-block;
+      width: 25px;
+      height: 10px;
+      border-radius: 32px;
+      background-color: var(--color-blue-ribbon);;
+     }
+     
+    &__text {
+    color: var(--color-text);
+      line-height: 150%;
+     }
+  }
+}
+
+@media screen and (max-width: 480px) {
+  .Reinforcement{
+    height: auto;
+    flex-direction: column;
+    margin: 5vw;
+    gap: 0;
+  }
+}
+    
```

Comment:
we might also add some media query for max-width 768px (currently it does not look good at such screen)

Reply:
maybe the easiest way is to change the margin values here (on the top of our css):
.Reinforcement {
  margin: 96px;
=> 0 auto;
plus: 
width: 90vw or % ?
just a suggestion..